src/hello-world.py

```diff
@@ -90,5 +90,5 @@
 
 def printLock():
     print(lock)
-
-printLock()
+if __name__ == '__main__':
```

Comment:
Ensure there's still an empty line between each top-level stanza:
```suggestion

if __name__ == '__main__':
```